misc/language-tour-guide/src/utils/content/navigation-builder.ts

```diff
@@ -84,6 +69,16 @@ export function getNodeNavigationSiblings(tree: NavigationTree, path: string) {
 
   traverse(tree);
 
+  const parentPath = path.split("/").slice(0, -1).join("/");
+
+  const children = flatNavigation.filter((item) => {
+    const itemParentPath = item.path.split("/").slice(0, -1).join("/");
```

Comment:
This doesn't handle query params and is pretty opaque in what it does. Can you add a function that parses it to URL, drops query params and does thing above?

We can also preprocess items if it isn't that much work so we don't do this filtering constantly.